docs/develop/scheduler-policy.md

```
@@ -132,17 +132,17 @@ So, in `Spread` policy we can select `Node2`.
 
 Binpack mainly focuses on the computing power and video memory usage of each card. The more it is used, the higher the score.
 ```
-score: ((request.core + used.core) / allocatable.core + (request.mem + used.mem) / allocatable.mem)) * 10
+score: ((request.core + used.core) / allocatable.core + (request.mem + used.mem) / allocatable.mem) * 10
 ```
 
 1. Binpack scoring information for GPU 1 is as follows
 ```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)